### PR TITLE
add a config item [security]spilled-file-encryption-method in tidb-configuration-file.md

### PR DESCRIPTION
**Pull request by mmyj**
### What is changed, added or deleted? (Required)

add the config item `[security]spilled-file-encryption-method`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/tidb/pull/19070 https://github.com/pingcap/docs-cn/pull/4302
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -269,6 +269,12 @@ Configuration items related to security.
 - The path of the SSL private key file used to connect TiKV or PD with TLS.
 - Default value: ""
 
+### `spilled-file-encryption-method`
+
++ Determines the encryption method used for saving the spilled files to disk。
```
Comment (XuHuaiyu) [inline tidb-configuration-file.md:274]: `。` --> `.`

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -269,6 +269,12 @@ Configuration items related to security.
 - The path of the SSL private key file used to connect TiKV or PD with TLS.
 - Default value: ""
 
+### `spilled-file-encryption-method`
+
++ Determines the encryption method used for saving the spilled files to disk.
++ Default value: `"plaintext"`, means disable encryption
```
Comment (TomShawn) [inline tidb-configuration-file.md:275]:
```suggestion
+ Default value: `"plaintext"`, which means to disable encryption.
```

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -269,6 +269,12 @@ Configuration items related to security.
...
++ Optional values: `"plaintext"`, `"aes128-ctr"`
```
Comment (TomShawn) [inline tidb-configuration-file.md:276]:
```suggestion
+ Optional values: `"plaintext"` and `"aes128-ctr"`
```

**Comment by TomShawn** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**